.circleci/config.yml

```diff
@@ -10,16 +10,14 @@ jobs:
   test:
     working_directory: *workspace-dir
     docker:
-      - image: circleci/python:3.8
+      - image: cimg/python:3.8
     steps:
       - attach_workspace:
           at: *workspace-dir
       - checkout
       - run:
           name: Install requirements
           command: |
-            rm `which poetry`
-            curl -sSL https://install.python-poetry.org | python -
             poetry lock
```

Comment:
what does this command do?
Usually, `poetry install --no-ansi` does the trick

Reply:
I think this command will create a lock file based on `pyproject.toml` file. But since we already have a `poetry.lock file`, we don't need this command.
[https://python-poetry.org/docs/cli/#lock](https://python-poetry.org/docs/cli/#lock)

Reply:
interesting. 
Can you point me to the failing workflow?

Reply:
sure I think is this one https://app.circleci.com/pipelines/github/up42/up42-py/3245/workflows/7df060c0-19d3-4267-a018-066fa99f97ef/jobs/3661